src/RulesTest/Library/BoundingRectangleSizeReasonableTests.cs

```diff
@@ -32,6 +32,18 @@ public void TestBoundingRectangleSizeReasonableEmptyFail()
                 Assert.IsFalse(Rule.PassesTest(e));
             } // using
         }
+        
+        [TestMethod]
```

Comment:
Maybe add a test where the ItemStatus check passes but the Framework check fails? Won't impact code coverage but just to pin the intent

Reply:
Good idea, done